mozilla_repo_urls/parser.py

```diff
@@ -21,7 +21,22 @@ def parse(url_string):
     if not parsed_url.valid:
         raise InvalidRepoUrlError(url_string)
 
-    if parsed_url.platform not in _SUPPORTED_PLAFORMS:
-        raise UnsupportedPlatformError(url_string, platform, _SUPPORTED_PLAFORMS)
-
+    if parsed_url.platform not in _SUPPORTED_PLATFORMS:
+        """
+        For error reporting purposes, the exception object includes the domain
+        for each supported platform.
+        """
```

Comment:
Question: I've never used triple double quotes for comments. What are the pros compared to a `# regular comment`? If I'm not mistaken, unlike a regular comment, this statement evaluates an unused string.

Reply:
It makes it easier to edit comments, since if line lengths change, then you don't need to adjust where each of the '#'s go, but it's not a big deal.